modular_skyrat/modules/microfusion/code/microfusion_energy_master.dm

```diff
@@ -1,6 +1,7 @@
 #define DUALWIELD_PENALTY_EXTRA_MULTIPLIER 1.4
 
 // Master file for cell loadable energy guns. PROCS ONLY YOU MONKEYS!
+// This file is a copy/paste of _energy.dm with extensive modification.
```

Comment:
Then this should be a subtype of energy guns. Why is this its own type?

Reply:
To maintain as much modularity as possible while also restricting interactions

Reply:
I'd have to override most of the procs anyway, and this protects it from upstream changes too, it was a subtype before I started running into a lot of proc override issues, so I started overriding base energy gun procs, but eventually decided upon this.